### ISSUE TITLE
Rewards setup panel is not translated

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#14368]
Follow up to https://github.com/brave/brave-browser/issues/13379

## Steps to Reproduce

1. Clean install
2. Run Brave in Polish (PL) locale

**Rewards Panel**
3. Open the rewards panel
4. Click "Start using Rewards"
5. Click `Let's take a quick tour` 
6. Go through the tour till Rewards setup panel is shown

**Rewards Page**
3. Navigate to `brave://rewards`
4. Click "Start using Rewards"
5. Click `Let's take a quick tour` 
6. Go through the tour till Rewards setup panel is shown

**Tip Dialog**
3. Navigate to `https://twitter.com/kamiljozwiak`
4. Click `Napiwek` (`Tip`) on a tweet
5. Click "Start using Rewards"
6. Click `Let's take a quick tour` 
7. Go through the tour till Rewards setup panel is shown

## Actual result:

Rewards setup panel is not translated

**Rewards Panel**
![image](https://user-images.githubusercontent.com/34715963/109182044-e1104a00-778c-11eb-8c7e-a16687f47b93.png)

**Rewards Page**
![image](https://user-images.githubusercontent.com/34715963/109184140-f71f0a00-778e-11eb-9e10-023ceb15becc.png)

**Tip Dialog**
![image](https://user-images.githubusercontent.com/34715963/109185279-284c0a00-7790-11eb-9917-c3602d0e74c8.png)

## Expected result:
Rewards setup panel is not translated

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

cc @brave/legacy_qa @rebron @mkarolin

**Comment by kjozwiak** [comment]
As per https://github.com/brave/brave-browser/issues/14369#issuecomment-788195579, moving issue into https://github.com/brave/brave-browser/milestone/167 now that https://github.com/brave/brave-core/pull/8093 was merged into `1.22.x` and `1.21.x` via https://github.com/brave/brave-core/pull/8096 & https://github.com/brave/brave-core/pull/8097.

@brave/legacy_qa if it's still an issue, please re-open and remove it from the milestone as there was no code associated with the issues so we can just re-open then and decide if it's release blocking or not.

**Comment by GeetaSarvadnya** [comment]
Verification passed on

```

Brave | 1.21.71 Chromium: 89.0.4389.72 (Offizieller Build) (64-Bit)
-- | --
Überarbeitung | 3f345f156bfd157bd1bea06310e55f3fb2490359-refs/branch-heads/4389@{#1393}
Betriebssystem | Windows 10 OS Version 2004 (Build 19041.804)

```
- Ensured Ads setup panel is translated in `Germany` and `France` locales

**Germany**

_BR panel_
![image](https://user-images.githubusercontent.com/38657976/109646890-aa687400-7b7e-11eb-99bc-113010a192b0.png)

_Rewards settings page_
![image](https://user-images.githubusercontent.com/38657976/109647004-d4219b00-7b7e-11eb-9546-b53fa9b7ea61.png)

_Tipping banner_
![image](https://user-images.githubusercontent.com/38657976/109647236-0cc17480-7b7f-11eb-8d3e-06807c185eef.png)

**France**
_BR panel_
![image](https://user-images.githubusercontent.com/38657976/109647480-56aa5a80-7b7f-11eb-83f6-5ce70c9077e0.png)

_Rewards settings page_
![image](https://user-images.githubusercontent.com/38657976/109647600-7f325480-7b7f-11eb-8493-e00900398c12.png)

_Tipping banner_
![image](https://user-images.githubusercontent.com/38657976/109647726-ac7f0280-7b7f-11eb-924a-3ae835e4fd99.png)
